beacon-chain/blockchain/service.go

```diff
@@ -140,6 +153,7 @@ func (c *ChainService) updateHead(processedBlock <-chan *pb.BeaconBlock) {
 			return
 		case block := <-processedBlock:
 			if block == nil {
+				c.failStatus = fmt.Errorf("chain head block not receieved for updating ")
```

Comment:
I don’t think this is necessarily an unhealthy condition. An empty thing sent through the channel can be ignored.

Also, is this failure permenant? What about when the service recovers?

Reply:
Understood something wrong. Agreed it will impact normal functioning. Will check these